db/migrations/20200128041603-create-package.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('packages', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      endDate: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      views: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      ratings: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
```

Comment:
If this stores multiple ratings, maybe a single integer isn't best suited for this. Let me know if I'm misinterpreting this.